include/openmc/kdtree_partitioner.h

```diff
@@ -0,0 +1,63 @@
+#ifndef OPENMC_KDTREE_PARTITIONER_H
+#define OPENMC_KDTREE_PARTITIONER_H
+
+#include "octree_partitioner.h"
+#include "partitioner_utils.h"
+#include "universe.h"
+#include "zplane_partitioner.h"
+#include <vector>
```

Comment:
git-clang-format sometimes gives me improper results. It might be better for someone to clone the repo after I've fixed everything else to run clang-format on their end.